src/Components/NewForm.vue

```diff
@@ -35,6 +35,31 @@
 			rows="4"
 			:placeholder="t('announcementcenter', 'Write announcement text, Markdown can be used …')" />
 
+		<div class="announcement__form__schedule">
+			<NcCheckboxRadioSwitch :checked.sync="scheduleEnabled">
+				{{ t('announcementcenter', 'Schedule Announcement Time (optional)') }}
+			</NcCheckboxRadioSwitch>
+			<NcDateTimePicker v-model="scheduleTime"
+				:disabled="!scheduleEnabled"
+				:clearable="true"
+				:disabled-date="disabledInPastDate"
+				:disabled-time="disabledInPastTime"
+				:showSecond="false"
```

Comment:
```suggestion
				:show-second="false"
```

Reply:
Oh, I am going to test, if this still works. I have seen the warning, but it was working with upper case